src/Shared/AWS/SemanticConventionVersion.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// because this class is shared and public, each project that exposes it must
+// ensure that it has a unique namespace, otherwise users could face type collisions
+// if they import multiple projects.
+#if Instrumentation_AWSLambda
+namespace OpenTelemetry.Instrumentation.AWSLambda;
+#elif Instrumentation_AWS
+namespace OpenTelemetry.Instrumentation.AWS;
+#elif Resources_AWS
+namespace OpenTelemetry.Resources.AWS;
+#endif
+
+#pragma warning disable SA1300
```

Comment:
Consider removing this pragma and `v1_27_0_Experimental` to `V1_27_0_Experimental`. It is still readable. but does not violate C# namign convention.

I do not see good possibility for the underscores.

Reply:
agreed - renamed enums and class names to match this convention.